Userland/Applications/SystemMonitor/main.cpp

```diff
@@ -220,8 +227,11 @@ int main(int argc, char** argv)
     process_table_view.set_key_column_and_sort_order(ProcessModel::Column::CPU, GUI::SortOrder::Descending);
     process_model->update();
 
+    Config::pledge_domains("SystemMonitor");
+    u32 frequency = Config::read_i32("SystemMonitor", "Monitor", "Frequency", 3);
```

Comment:
You're reading an i32 value into a u32 variable, if this is intended, please do a static cast to show this.

Reply:
Changed it to i32.